src/ZoneServer/World/Actors/Monsters/Mob.cs

```diff
@@ -381,47 +394,132 @@ private void DropItems(Character killer)
 
 			var rnd = RandomProvider.Get();
 			var autoloot = killer?.Variables.Temp.Get("Autoloot", 0) ?? 0;
+			var worldConf = ZoneServer.Instance.Conf.World;
 
-			foreach (var dropItemData in this.Data.Drops)
-			{
-				var dropChance = GetAdjustedDropRate(dropItemData);
+			var iterations = 1;
 
-				var dropSuccess = rnd.NextDouble() < dropChance / 100f;
-				if (!dropSuccess)
-					continue;
+			// these should never be simultaneously applied, but just in case we apply gold last
+			if (this.IsBuffActive(BuffId.EliteMonsterBuff)) iterations = worldConf.EliteRolls;
+			if (this.IsBuffActive(BuffId.SuperDrop)) iterations = worldConf.SilverJackpotRolls;
+			if (this.IsBuffActive(BuffId.TwinkleBuff)) iterations = worldConf.GoldJackpotRolls;
 
-				if (!ZoneServer.Instance.Data.ItemDb.TryFind(dropItemData.ItemId, out var itemData))
+			for (int i = 0; i < iterations; i++)
```

Comment:
I guess a roll is something of a drop multiplier then 🤔 Though you're also calling them "iterations". I feel like a more descriptive name would be good here. Even something not entirely accurate, like `SilverJackpotDropMultiplier`, might be easier to understand than "roll" or "interation" in my mind.